lib/pages/diets_page.dart

```diff
@@ -11,30 +14,141 @@ import '../utility/rr_buttons.dart';
+                              top: constants.paddingUnit / 2, bottom: constants.paddingUnit / 2),
+                          decoration: BoxDecoration(
+                            border: Border.all(
+                                color: Theme.of(context).colorScheme.surface,
+                                width: constants.borderWidth),
+                            borderRadius: BorderRadius.circular(constants.dInnerRadius),
+                          ),
+                          child: ListTile(
+                              contentPadding:
+                                  EdgeInsets.only(left: constants.paddingUnit, right: 0),
+                              title: StyledHeadline(
+                                  text: gagList[index].date,
+                                  textStyle: Theme.of(context).textTheme.headlineSmall!.copyWith(
+                                        fontWeight: FontWeight.bold,
+                                        height: 1,
+                                        leadingDistribution: TextLeadingDistribution.proportional,
+                                      )),
+                              onTap: () {},
+                              trailing: gagList[index].familyRatio.isEmpty &&
+                                      gagList[index].soloRatio.isEmpty
+                                  ? SizedBox(
+                                      height: double.infinity,
+                                      child: TextButton(
+                                          child: const Icon(
+                                            Icons.add_circle_outline,
+                                            color: Colors.black,
+                                          ),
+                                          onPressed: () {
+                                            showInputDialog(context, index);
+                                          }))
+                                  : SizedBox(
+                                      height: double.infinity,
+                                      child: TextButton(
+                                          child: const Icon(
+                                            Icons.arrow_forward_ios_rounded,
+                                            color: Colors.black,
+                                          ),
+                                          onPressed: () {
+                                            Navigator.push(
+                                              context,
+                                              MaterialPageRoute(
+                                                  builder: (context) => DailyDishesScreen(index)),
+                                            );
+                                          })),
+                              subtitle: Wrap(
+                                  spacing: constants.paddingUnit / 2,
+                                  runSpacing: -constants.paddingUnit,
```

Comment:
Если закинуть в стиль `ElevatedButton` ещё строчку `tapTargetSize: MaterialTapTargetSize.shrinkWrap`, то противный стандартный паддинг у кнопки уберётся и можно будет написать православное 
```suggestion
                                  runSpacing: constants.paddingUnit / 2
```

Reply:
оо, круто, спасибо!